src/js/video.js

```diff
@@ -235,7 +235,7 @@ class Video {
 
 		this.containerEl.appendChild(this.videoEl);
 
-		addEvents(this, ['playing', 'pause', 'ended', 'progress', 'seeked']);
+		addEvents(this, ['playing', 'pause', 'ended', 'progress', 'seeked', 'error', 'stalled']);
```

Comment:
@i-like-robots Could you add a test for these new event listeners, it would be around this line -- https://github.com/Financial-Times/o-video/blob/master/test/video.test.js#L134